posthog/temporal/batch_exports/bigquery_batch_export.py

```diff
@@ -354,6 +354,8 @@ async def flush_to_bigquery(bigquery_table, table_schema):
 
                     jsonl_file.reset()
 
+        return jsonl_file.records_total
```

Comment:
This is de-dented outside of the `with BatchExportTemporaryFile() as jsonl_file:`, while it may work (which surprises me, but Python loves to leak things from scope) I think we should put it inside?

Reply:
Yeah, with statements do not introduce start a new scope as a function does. I am fine with de-denting this though, and I do plan to address it more in future PRs.